geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1029,12 +1032,23 @@ def call(fnames, command_line_args=None):
             raise ValueError("Need to set both $GEOIPS_DB_URI")
 
     pid_track.track_resource_usage(logstr="MEMUSG", verbose=False, key="READ METADATA")
-    reader_plugin = readers.get_plugin(config_dict["reader_name"])
+    if command_line_args.get("reader_name"):
```

Comment:
Does this need tobe comented?

Reply:
I think this is pretty self-evident. It just allows overriding the particular reader at the CLI.